messages/pt.json

```diff
@@ -97,5 +97,6 @@
   "store/reviews.list.pagination.textOf": "de",
   "store/reviews.list.graph.stars": "{value, plural, =0{0 estrelas} =1{1 estrela} other{# estrelas}}",
   "store/reviews.summary.add.login": "Por favor faça login para escrever avaliação",
-  "store/reviews.summary.add.title": "Adicionar avaliação"
-}
+  "store/reviews.summary.add.title": "Adicionar avaliação",
+  "store/reviews.list.loadingReviews": "Carregando resenhas ..."
```

Comment:
The correct translation is "Carregando avaliações..."

Reply:
Thanks, i'll update